### ISSUE TITLE
Sigmoid not used?

### ISSUE THREAD

**Issue by petwill** [tkipf/gae#21]
Hi @tkipf 
Seems like a simple linear activation is used for the decoder?
If yes, why isn't sigmoid used? (as mentioned in the paper

Thanks in advance!

**Comment by petwill** [comment]
Sorry my mistake, saw that in `tf.nn.weighted_cross_entropy_with_logits`
